Ginger/GingerCoreCommon/Repository/ApplicationModelLib/ApplicationModelBase.cs

```diff
@@ -47,6 +47,11 @@ public abstract class ApplicationModelBase : RepositoryItemBase
             }
         }
 
+        public override string ToString()
```

Comment:
for what this is needed?

Reply:
To display nice name when we drag the API Folder. Currently it displays complete class path as, "Amdocs.Ginger.Repository.RepositoryFolder".